lib/libsys/intro.2

```diff
@@ -738,6 +738,11 @@ for a specific group of protocols.
 Each socket has an address
 chosen from the address family in which the socket was created.
 .El
+.Sh FILES
+.Bl -inset -compact
+.It Pa /usr/include/sys/syscall.h
```

Comment:
I wonder if we should add `.In sys/syscall.h` to SYNOPSIS as well?

Reply:
May I do it now as part of the commit adding syscall.h to FILES or should I make a future PR?

Should `.In errors.h` also be made to say `.In sys/errors.h`?

Reply:
Hmmm, should the FILES section also have `/usr/include/sys/errors.h`?